org.flameshot.flameshot.yml

```diff
@@ -0,0 +1,33 @@
+app-id: org.flameshot.flameshot
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: flameshot
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+  # Connectivity
+  - --share=network
+  # QtSingleApplication, allow other instances to see log files
+  - --env=TMPDIR=/var/tmp
+  # Allow loading/saving files from anywhere
+  - --filesystem=host
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  # System Tray Icon
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-1
```

Comment:
This won't work.

Reply:
Can you provide more clarity on which part won't work?

Reply:
the numbers at the end are the PID. Flatpak apps have sandboxed PIDs so they can collide with PIDs from other apps or the system, not a problem if that ID stays inside the sandbox. Once you rely on poking a hole into the rest of the system and using that ID as a unique value though, things can go wrong. The more flatpak apps you use that use the KDE statusnotifier protocol, the more likely this is to happen.

Reply:
Okay thanks for clarifying, ill figure out a way to fix it.